src/shrub/js/grade/grade.js

```diff
@@ -25,6 +26,10 @@ export function Remove( node, grade ) {
 	return Fetch.Get(API_ENDPOINT+'/vx/grade/remove/'+node+'/'+grade, true);
 }
 
+export function GetGameList( node ) {
+	return Fetch.Get(API_ENDPOINT+'/vx/grade/getgames/' + node, true);
```

Comment:
Endpoint should be changed to be `getmylist` (in PHP too).

Reply:
Fixed (in php too)